### DISCUSSION TITLE
Smooth upgrade to Debian 12

### DISCUSSION THREAD

**Discussion by yasershahi** [SpiralLinux/SpiralLinux-project#282] [General]
I want to say thank you for this project. I installed SpiralLinux and it upgraded to Debian 12 with no issues at all. Fantastic font rendering and polished desktop experience that makes Debian a rock solid daily driver!

**Comment by methanoid** [comment]
@geckolinux I like to  use command line... is there some special way to "sudo apt update && sudo apt upgrade" to Bookworm from Bullseye having changed the contents of /etc/apt/sources.list that avoids the need for interaction (it asks you to decide if you want to keep some configs or to restart some services) so you cannot set it going and walk away for 10m while it upgrades you???

**Reply by geckolinux** [reply]
I wouldn't recommend it, because it's best to keep some of the configs, for example the PulseAudio config comes to mind.

**Comment by methanoid** [comment]
Yes, the default IS to keep those configs (I just did an install about 30m ago) so I am wondering if there is an option to "do the defaults and leave me alone please" ?? :-)
